app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -30,7 +30,7 @@ public static SessionStore get() {
         }
         return mInstance;
     }
-    public static final String DEFAULT_URL = "resource://android/assets/html/index.html";
+    public static final String HOME_NO_REGION_ORIGIN = "https://webxr.today";
```

Comment:
note: this is pointing to our staging server (which reflects [`master` of the `ContentFeed` repo](https://github.com/mozillareality/contentfeed/tree/master)). expect the domain to change very soon. it's just a placeholder for now.